src/components/Notification/LoadingAnimation.jsx

```diff
@@ -20,7 +21,7 @@ import Typography from '@mui/material/Typography';
  * @param {loadingAnimationProps} Props - Component props for LoadingAnimation
  * @returns {React.ReactElement} a div of what is currently loading
  */
-const LoadingAnimation = ({ loadingItem }) => (
+const LoadingAnimation = ({ loadingItem, animationType = 'linear' }) => (
```

Comment:
How about instead of passing in an `animationType`, we just let consumers pass in an animation component?

```
const LoadingAnimation = ({loadingItem, children}) => {
  <Box>
    <Paper>
      <Typography> Loading {loadingItem}... </Typography>
      {children ? children: <LinearProgress />}
    </Paper>
  </Box>
}

...

const LoadingAnimationConsumer = () => (
  <LoadingAnimation>
    <CircularProgress />
  </LoadingAnimation>
)
```

Reply:
Yeah, think that would be a good idea.

Reply:
Got this to take in a animation component as a children with LinearProgress as default if no children are passed in.